src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -22,10 +22,14 @@ def update
                       fetch_notifications.where(id: params[:notification_ids])
                     end
     # rubocop:disable Rails/SkipsModelValidations
-    unless notifications.update_all('delivered = !delivered')
+    updated_notifications = notifications.update_all('delivered = !delivered')
+    # rubocop:enable Rails/SkipsModelValidations
+
+    if updated_notifications.zero?
       flash.now[:error] = "Couldn't mark the notifications as #{notifications.first.unread? ? 'read' : 'unread'}"
     end
-    # rubocop:enable Rails/SkipsModelValidations
+
+    send_notifications_information_rabbitmq(updated_notifications.size) if updated_notifications.zero?
```

Comment:
```
[10] pry(main)> 1.size
=> 8
[11] pry(main)> 6.size
=> 8
[12] pry(main)> 9382721.size
=> 8
```

Reply:
ah no right, I was expecting an array of elements changed, not a number.